curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-an-ordered-list.english.md

```diff
@@ -42,6 +42,10 @@ tests:
     testString: assert(code.match(/<\/ol>/g) && code.match(/<\/ol>/g).length === code.match(/<ol>/g).length, 'Make sure your <code>ol</code> element has a closing tag.');
   - text: Make sure your <code>li</code> element has a closing tag.
     testString: assert(code.match(/<\/li>/g) && code.match(/<li>/g) && code.match(/<\/li>/g).length === code.match(/<li>/g).length, 'Make sure your <code>li</code> element has a closing tag.');
+  - text: The <code>li</code> elements in your unordered list should not be empty.
+    testString: $('ul li').each((i, val) => assert(!!val.textContent.replace(/\s/g, '').length, 'Your <code>li</code> elements in your unordered list should not be empty.'));
```

Comment:
Don't need double negation/NOT (`!!`). `.length` will be a valid number and only falsy value could be `0` which is falsy anyway.